CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+**Added:**
```

Comment:
Please also add a "Changed" section for the executable file, and notify the users that they might need to install a new version of runtime for running the model via exe file in this version.